server/modules/surveyRdb/service/surveyRdbService.js

```diff
@@ -65,12 +65,12 @@ const queryTable = async (surveyId, nodeDefUuidTable, tableName, nodeDefUuidCols
       async row => {
         const { record_uuid: recordUuid } = row
 
+        const record = await RecordManager.fetchRecordByUuid(surveyId, recordUuid)
+
         const resultRow = {
           ...row,
           cols: {},
-          record: {
-            uuid: recordUuid,
-          },
+          record,
```

Comment:
is it really needed to pass all the record? I think it's better to pass only the validation status. or?

Reply:
right